pkg/util/names/names.go

```diff
@@ -134,3 +137,14 @@ func GeneratePolicyName(namespace, name, gvk string) string {
 
 	return strings.ToLower(fmt.Sprintf("%s-%s", name, rand.SafeEncodeString(fmt.Sprint(hash.Sum32()))))
 }
+
+var rbacKindStrings = sets.NewString("Role", "ClusterRole", "RoleBinding", "ClusterRoleBinding")
```

Comment:
Can you explain why we need the preset list?
It looks weird to me because for those resource types that don't support `:` in name, they never get a chance to be applied in karmada-apiserver, and for resource types that support `:` in the name(other than RBAC resources, if there is any), why don't we support them?

Reply:
It seems superfluous, I remove this list.